src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset id="first" class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            autocomplete="off"
+            name="surname"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            type="text"
+            autocomplete="off"
+            name="name"
+            maxlength="15"
+          >
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            id="age"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="age"
+          >
+        </div>
+        <div class="form-field">
+          <label for="date-of-birth">Full date of birth:</label>
+          <input
+            id="date-of-birth" type="date"
+            placeholder="mm/dd/yyyy" name="date-of-birth"
+          >
```

Comment:
[checklist p3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
          <input
            id="date-of-birth"
            type="date"
            placeholder="mm/dd/yyyy"
            name="date-of-birth"
          >
```